### PR TITLE
update azure_blobDiskController log format

### PR DESCRIPTION
**Pull request by lovebaby979**
```docs
update azure_blobDiskController log format
```

### CODE REVIEW

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func (c *BlobDiskController) initStorageAccounts() {
 		// get accounts
 		accounts, err := c.getAllStorageAccounts()
 		if err != nil {
-			klog.Errorf("azureDisk - getAllStorageAccounts error: %v", err)
+			klog.ErrorS(nil, "azureDisk - getAllStorageAccounts error", "error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go:77]:
```suggestion
			klog.ErrorS(nil, "azureDisk - getAllStorageAccounts error", "err", err)
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go [hunk 2]
```diff
@@ -127,7 +127,7 @@ func (c *BlobDiskController) DeleteVolume(diskURI string) error {
 	}
 	err = c.common.cloud.deleteVhdBlob(accountName, key, blob)
 	if err != nil {
-		klog.Warningf("azureDisk - failed to delete blob %s err: %v", diskURI, err)
+		klog.InfoS("azureDisk - failed to delete blob err", "deleteBlobURI", diskURI, "error", err)
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go:130]:
```suggestion
		klog.InfoS("azureDisk - failed to delete blob err", "deleteBlobURI", diskURI, "err", err)
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go [hunk 3]
```diff
@@ -198,8 +198,8 @@ func (c *BlobDiskController) createVHDBlobDisk(blobClient azstorage.BlobStorageC
 		End:   uint64(vhdSize - 1),
 	}
 	if err = blob.WriteRange(blobRange, bytes.NewBuffer(h[:vhd.VHD_HEADER_SIZE]), nil); err != nil {
-		klog.Infof("azureDisk - failed to put header page for data disk %s in container %s account %s, error was %s\n",
-			vhdName, containerName, accountName, err.Error())
+		klog.InfoS("azureDisk - failed to put header page for data disk  in container account",
+			"dataDiskName",vhdName, "containerName", containerName, "accountName", accountName, "error", err.Error())
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go:202]:
```suggestion
			"dataDiskName",vhdName, "containerName", containerName, "accountName", accountName, "err", err.Error())
```

File: staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go [hunk 4]
```diff
@@ -380,15 +381,15 @@ func (c *BlobDiskController) ensureDefaultContainer(storageAccountName string) e
 			_, provisionState, err := c.getStorageAccountState(storageAccountName)
 
 			if err != nil {
-				klog.V(4).Infof("azureDisk - GetStorageAccount:%s err %s", storageAccountName, err.Error())
+				klog.V(4).InfoS("azureDisk - GetStorageAccount", "storageAccountName", storageAccountName, "error", err.Error())
```
Comment (yangjunmyfm192085) [inline staging/src/k8s.io/legacy-cloud-providers/azure/azure_blobDiskController.go:384]:
```suggestion
				klog.V(4).InfoS("azureDisk - GetStorageAccount", "storageAccountName", storageAccountName, "err", err.Error())
```

**Review by feiskyer — COMMENTED** [review]
let's keep the legacy code as is because only bug fixes are allowed now for legacy codes.

/close

**Comment by feiskyer** [comment]
@lovebaby979 thanks for the contribution!

**Pull request closed without merging**